crates/egui/src/containers/combo_box.rs

```diff
@@ -371,7 +374,9 @@ fn combo_box_dyn<'c, R>(
                     // result in labels that wrap very early.
                     // Instead, we turn it off by default so that the labels
                     // expand the width of the menu.
-                    ui.style_mut().wrap = Some(false);
+                    if ui.style().wrap_mode == Some(TextWrapMode::Wrap) {
+                        ui.style_mut().wrap_mode = None;
+                    }
```

Comment:
The comment no longer matches the behavior.

Why isn't this just 
```suggestion
                    ui.style_mut().wrap_mode = Some(TextWrapMode::Extend);
```
?